src/services/email.services.js

```diff
@@ -0,0 +1,88 @@
+import nodemailer from 'nodemailer';
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+}
```

Comment:
Consider adding error handling for the send function to manage cases where sending an email fails.